docs/getting-started/installation.md

```diff
@@ -49,9 +52,9 @@ In this section, you can read the [System Requirements](/getting-started/require
         bbdd.sid=etendo
         bbdd.port=5432
         bbdd.systemUser=postgres
```

Comment:
If you changed "systemPassword" to "system_user_password", change also systemUser to "system_user"